app/assets/stylesheets/active_admin/_forms.scss

```diff
@@ -134,6 +141,10 @@ form {
     }
   }
 
+  input[type=date] {
+    width: calc(100% - #{$text-input-total-padding});
+  }
```

Comment:
Ok, I think I know what you mean. You can take a look, but if you don't found something, keep your fix.